### PR TITLE
[ui] fix for currency pair input retaining global state

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

File: ui/app/src/components/currencyPairPanel/Index.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <template>
   <div class="field-wrappers">
     <CurrencyField
-      label="Input"
+      label="To"
```
Comment (ryardley) [inline ui/app/src/components/currencyPairPanel/Index.vue:6]: Hmm I think we're gonna need to pass these labels in to the `CurrencyPairPanel` and give them defaults of Input.

**Review by ryardley — CHANGES_REQUESTED** [review]

File: ui/app/src/components/currencyPairPanel/Index.vue (hunk 1 above)
Comment (thomasdavis) [reply ui/app/src/components/currencyPairPanel/Index.vue:6]: Will fix this now

File: ui/app/src/hooks/useCurrencyFieldState.ts [hunk 2]
```diff
@@ -18,15 +16,16 @@ type CurrencyFieldState = {
 export function useCurrencyFieldState(): CurrencyFieldState {
   // Copy global state when creating page state
   const fromSymbol = ref<string | null>(globalState.fromSymbol.value);
-  const fromAmount = ref<string>(globalState.fromAmount.value);
   const toSymbol = ref<string | null>(globalState.toSymbol.value);
-  const toAmount = ref<string>(globalState.toAmount.value);
+
+  // Local page state
```
Comment (59023g) [inline ui/app/src/hooks/useCurrencyFieldState.ts:21]: I like comments

**Review by 59023g — APPROVED** [review]
💯

**Review by ryardley — APPROVED** [review]
💯

**Review by ryardley — APPROVED** [review]
💯 🤘

**Pull request merged**